src/pages/index.js

```diff
@@ -57,37 +60,82 @@ const previewCloseButton = previewModal.querySelector(".modal__close-btn");
 
 const previewCaption = previewModal.querySelector(".modal__caption");
 
+const deleteModal = document.querySelector("#delete-modal");
+
+const deleteForm = document.forms[2];
+
+const cancelButton = deleteModal.querySelector(".modal__delete-btn_cancel");
+
+const closeDeletebutton = deleteModal.querySelector(".modal__close-btn");
+
+const avatarModal = document.querySelector("#avatar-modal");
+
+const avatarFormElement = document.forms[3];
```

Comment:
`document.forms[3];`  

If the order of the forms change in `index.html`, your code will not work anymore because `[2]` will return another form


You should take the forms, using their `name` attribute value


```js
const avatarFormElement= document.forms["avatar-form"];


```